api/app_analytics/tasks.py

```diff
@@ -109,15 +109,16 @@ def track_feature_evaluation(
 
 @register_task_handler()
 def track_request(
-    resource: str,
+    resource: int,
```

Comment:
Just highlighting that this is the fix since it reverts the change of type as implemented [here](https://github.com/Flagsmith/flagsmith/pull/5330).